content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,67 @@
 Invoice Time Spent on Tickets (Postpaid Support Services)
 =========================================================
 
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect your Helpdesk ticket to a Project task and use either one to record timesheets that can be
+later pulled from to bill your client. This integration provides more control and transparency over
```

Comment:
Wording suggestion.
This is the first line of the doc and I had to reread it a couple times to make sense of it. Technically it's correct but it's clunky...can clean up a little.
```suggestion
Connect your Helpdesk ticket to a Project task, and use either one to record timesheets that can
later be pulled from in order to bill the client. This integration provides more control and
transparency over
```

Reply:
I changed the wording and structure a little to be more like chronological steps. Hopefully, that reads smoother!